README.md

```diff
@@ -4,7 +4,94 @@ This project implements a Matter Controller Server over WebSockets using the [of
 
 The goal of this project is primary to have Matter support in Home Assistant but its universal approach makes it suitable to be used in other projects too.
 
-This repository is for development only (so not for enduser support). For enabling Matter support within Home Assistant, please refer to the [Home Assistant documentation](https://www.home-assistant.io/integrations/matter/).
```

Comment:
we should somehow keep this text in as this still applies, this is the repo for the matter server library and for HA enduser support, people should go to HA.

Reply:
The reason I've removed it is that the same readme is shown in the package repository. So this could be interpreted somewhat wrong.